src/models/CarType.ts

```diff
@@ -1 +1 @@
-export const carTypeList = ["sedan", "SUV", "hatchback"];
\ No newline at end of file
+export const carTypeList = ["sedan", "SUV", "hatchback", ""];
```

Comment:
what's the empty state?

Reply:
btw, there is no shame in taking a different config/constant when solving for a different use case.
Hint: you can also reuse the same constant(carTypeList) to create a new constant that can be used in carList page

Reply:
empty state is for view all cars tab,
its not affecting any other use case so placed in the same array as we are using specific index